policies/queries/apigateway/api_keys_disabled.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'             as execution_time,
+       :'framework'                  as framework,
+       :'check_id'                   as check_id,
+       'Unused API Gateway API keys' as title,
```

Comment:
I think this should be singular because you are only highlighting a single resource... I think this applies to all policies